vector/src/main/java/im/vector/app/features/settings/VectorPreferences.kt

```diff
@@ -227,7 +227,6 @@ class VectorPreferences @Inject constructor(
         private const val DID_ASK_TO_ENABLE_SESSION_PUSH = "DID_ASK_TO_ENABLE_SESSION_PUSH"
 
         const val TCHAP_SETTINGS_HIDE_FROM_USERS_DIRECTORY_PREFERENCE_KEY = "TCHAP_SETTINGS_HIDE_FROM_USERS_DIRECTORY_PREFERENCE_KEY"
-        const val TCHAP_SETTINGS_CHRISTMAS_PREFERENCE_KEY = "TCHAP_SETTINGS_CHRISTMAS_PREFERENCE_KEY"
```

Comment:
should not be part of this PR, it messes with commit history and overall understanding